devDocs/technicalDesignOverview.md

```diff
@@ -67,6 +67,14 @@ Aside from accessibility and native APIs, Windows provides many functions which
 Information that can be obtained includes the class name of a window, the current foreground window and system battery status.
 Tasks that can be performed include moving/clicking the mouse and sending key presses.
 
+### Logging
+
+#### Logging in secure mode
+`logHandler.initialize` prevents logging in secure mode.
+This is because it is a security concern to log during secure mode (e.g. passwords are logged).
+To change this for testing, patch the source build.
```

Comment:
I asked myself a similar question when reading this text: why do not use serviceDebug in registry?

Reply:
A PR to beta to amend this might be worthwhile.
I don't think this is misleading enough to patch 2021.3.4.
Using `serviceDebug` is still a documented option in the `userGuide`.
The important part of the patch is that you cannot enable debug logging in secure mode without this.
The security advisory, when published, should make the security implications clear of this patch,

```suggestion
To change this for testing, use the [serviceDebug](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#SystemWideParameters) system wide parameter.
```